Update spanish.json

MauricioLanda opened the pull request:
typo at line 709: serveral instead of several

Zenrac commented on i18n/spanish.json:
```diff
@@ -702,34 +702,34 @@
+    "cmd-blindtest-mal-same-time" : "Solo puedes iniciar {} listas de anime al mismo tiempo.",
+    "cmd-blindtest-max-animelist" : "No puedes poner esta cantidad de listas de anime. (max es 5)",
+    "cmd-blindtest-empty" : "Esta lista de reproducción automática está vacía! Se requiere al menos una canción para iniciar una prueba a ciegas ...",
+    "cmd-blindtest-help" : "Inicia una prueba a ciegas en el canal actual.\n\nOpciones Disponibles:\n\nlongestword=true|false (Default to False):\nAceptar la palabra más larga como una buena respuesta.\n\nlistening=true|false (Default to False):\n No acepta respuestas, solo escucha las canciones.\n\ntimeout=number (Default to 120):\n Tiempo de espera antes de detener la canción si no se encuentra ...
```
Do not translate the available status as they have to be written in english only

MauricioLanda commented:
Ok, i will fix that in a few hours and let you know with another PR

On Tue, May 7, 2019, 10:24 Zenrac <notifications@github.com> wrote:

> *@Zenrac* commented on this pull request.
> ------------------------------
>
> In i18n/spanish.json
> <https://github.com/Zenrac/watora-translations/pull/156#discussion_r281655952>
> :
>
> > -    "cmd-blindtest-empty" : "This autoplaylist is empty! It requires at least one song to start a blindtest...",
>
> -    "cmd-blindtest-help" : "Starts a blindtest in the current channel.\n\nAvailable options:\n\nlongestword=true|false (Default to False):\n Accept the longest word as a good answer.\n\nlistening=true|false (Default to False):\n Do not accept answers, just listen the songs.\n\ntimeout=number (Default to 120):\n Timeout before stopping the song if not found. Between 3 and 300 seconds.\n\nseverity=number (Default to 100):\n Severity in percentage of word to accept an answer. Between 1 and 100%.\n\npercentage=number,number,number (Default to 100,0,0):\n Percentage of OP, ED, OST. The sum of all values should be 100.\n\nwait=number (Default to 5):\n Time to wait before each song. Between 0 and 30 seconds.\n\nargument=status (Default to all):\n Only select a type of watched anime in the given anime lists.\nAvailable status are: all, completed, watching, onhold, dropped, ptw.\n\nremove=status,status2 (Default to ptw):\n Ignore a type (or serveral) of watched anime (separated by commas) in the given anime lists.\nAvailable status are: completed, watching, onhold, dropped, ptw.",
>
> -    "cmd-blindtest-is-running" : "A blindtest is already running.. Do you want to stop it ? (yes/no)",
>
> -    "cmd-blindtest-stop-blindtest" : "Stopping blindtest!",
>
> -    "cmd-blindtest-invalid-param" : "The parameter {} isn't valid.",
>
> -    "cmd-blindtest-invalid-value" : "The value {} isn't valid for the parameter {}.",
>
> -    "cmd-blindtest-blindtest-info" : "Fetching information from MyAnimeList...",
>
> -    "cmd-blindtest-fail" : "Fail",
>
> -    "cmd-blindtest-start" : "Starting a blindtest game in this channel with {} songs!",
>
> -    "cmd-blindtest-mal-fail" : "Failed to fetch information from MyAnimeList, try again later!",
>
> -    "cmd-blindtest-timeout" : "Timeout",
>
> +    "cmd-blindtest-no-enough-param" : "No se han dado suficientes parámetros para iniciar una prueba a ciegas ...",
>
> +    "cmd-blindtest-mal-same-time" : "Solo puedes iniciar {} listas de anime al mismo tiempo.",
>
> +    "cmd-blindtest-max-animelist" : "No puedes poner esta cantidad de listas de anime. (max es 5)",
>
> +    "cmd-blindtest-empty" : "Esta lista de reproducción automática está vacía! Se requiere al menos una canción para iniciar una prueba a ciegas ...",
>
> +    "cmd-blindtest-help" : "Inicia una prueba a ciegas en el canal actual.\n\nOpciones Disponibles:\n\nlongestword=true|false (Default to False):\nAceptar la palabra más larga como una buena respuesta.\n\nlistening=true|false (Default to False):\n No acepta respuestas, solo escucha las canciones.\n\ntimeout=number (Default to 120):\n Tiempo de espera antes de detener la canción si no se encuentra. Entre 3 y 300 segundos.\n\nseverity=number (Default to 100):\n Severidad en porcentaje de palabra para aceptar una respuesta. Entre 1 y 100%.\n\npercentage=number,number,number (Default to 100,0,0):\n Porcentaje de OP, ED, OST. La suma de todos los valores debe ser 100.\n\nwait=number (Default to 5):\n Tiempo para esperar antes de cada canción. Entre 0 y 30 segundos.\n\nargument=status (Default to all):\n Solo selecciona un tipo de anime visto en las listas de anime dadas.\nLos estados disponibles son: todos, completado, observación, retener, descartado, ptw.\n\nremove=status,status2 (Default to ptw):\n Ignora un tipo (o varios) de anime visto (separados por comas) en las listas de anime dadas.\nLos estados disponibles son: completado, observación, retención, descartado, ptw.",
>
>
> Do not translate the available status as they have to be written in
> english only
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Zenrac/watora-translations/pull/156#pullrequestreview-234547907>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACSQY66FRS226PRPJNDDAHDPUGGITANCNFSM4HLJHRNQ>
> .
>

Zenrac commented:
do not open another pull request, just edit your file in your fork, it will auto-update this pull request.

MauricioLanda commented:
Very well, thank you

On Tue, May 7, 2019, 10:27 Zenrac <notifications@github.com> wrote:

> do not open another pull request, just edit your file in your fork, it
> will auto-update this pull request.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Zenrac/watora-translations/pull/156#issuecomment-490103495>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACSQY62ERW7EFO3A65TKVW3PUGGWHANCNFSM4HLJHRNQ>
> .
>

MauricioLanda commented:
it is now changed

Zenrac commented:
Thanks
